maventa/MaventaService.ts

```diff
@@ -0,0 +1,73 @@
+function isMaventaInvoice(data: any): data is MaventaInvoice {
```

Comment:
We should never have `any` in the production code, nor any other part either (some times in special case maybe, but `unknown` is better for those usually.)

We should have specific types for everything. Otherwise the code base will get confusing very fast, and we will see decrease in the development time.

Reply:
And once this has a type instead of `any` the function probably should be moved to the type's file.